---
layout: default
title: Research 
categories: 
 - research
---

<div class="bigspacer"></div>
<div class="row">
	<div class="col-md-12">
		<h2>Background</h2>
		<div class="row">
			<div class="col-md-8">
				<p>Environment is rarely static -- temperatures rise and fall, food availability changes -- all demanding an organism to sense its surroundings and make proper responses. Sometimes, however, the environment may change dramatically or experience steady long-term shifts, such that the existing stress responses are no longer fit for the task. The <strong>goal of our lab</strong> is to understand how gene regulatory networks for stress responses evolve and how it may have contributed to species adaptation to its environment.</p>
				<p>To achieve this goal, we study a group of yeast species, which include the baker's yeast, <em>S. cerevisiae</em>, a related commensal species <em>C. glabrata</em>, which is also an opportunistic pathogen, as well as many others.</p>
			</div>
			<div class="col-md-4 col-xs-0">
				<img src="/images/stress-response-regulation-cartoon.png"/>
			</div>
		</div>
	</div>
	<div class="col-md-8">

		<h2>Specific aims</h2>

		<p>Our current research is focused on the following specific questions:</p>

		<ol>
			<li><h4>How does the structure of stress response networks evolve between related species?</h4>
				<p>We will comprehensively map the gene regulatory networks for major stress responses in commensal and related free-living species. Using conserved master transcription factors for each stress as "handles", we aim to uncover the underlying target network and compare them both at the level of network structure and downstream functions.</p>
			</li>
			<li><h4>What types of changes (protein, promoters or chromatin states), and in which part of the network, underlie the evolution of stress response networks?</h4>
				<p>As an example, our previous work in the Phosphate Starvation Response network identified the master transcription factor Pho4 to have evolved and acquired additional targets in the commensal yeast <em>C. glabrata</em>. We will study how protein sequence changes in Pho4 led to the expanded target set.</p>
			</li>
			<li><h4>How might the changes identified above contribute to species adaptation to their environment?</h4>
				<p>Not all changes between species are adaptive. Most, in fact, are results of "neutral evolution". To identify those changes that may have been involved in species adaptation, we will investigate the physiological and fitness consequences of the regulatory network changes. We also apply evolutionary sequence analyses to identify signatures of selection in both protein coding and regulatory DNA.</p>
			</li>
		</ol>

		<h2>Approaches</h2>
		<p>functional genomics, e.g. RNA-seq and ChIP-seq; molecular evolution (DNA sequence analysis) and phylogenetic approaches; phenotypic and fitness assays</p>

		<h2>Relevance for human health</h2>
		<p>By dissecting the differences in their stress response regulation and the underlying evolutionary history, we hope to better understand the challenges faced by commensal microbes and the strategies that proved successful by natural selection to allow them survive and thrive in that environment. This will lead to new perspectives on the relationship between host and commensals, including when and why they switch to the pathogenic mode -- a stress response from the microbe's perspective -- and will likely give rise to new theraputics and preventative measures for fungal infections.</p>
	</div>
</div>
